src/utils/actions/getActionsFromValues.js

```diff
@@ -0,0 +1,45 @@
+const docker = require('KegDocCli')
+const { loadValuesFiles } = require('KegConst/docker/loaders')
+const { generalError } = require('KegUtils/error/generalError')
+const { getSetting } = require('KegUtils/globalConfig/getSetting')
+const { getKegContext } = require('KegUtils/getters/getKegContext')
+
+/**
+ * Gets the action object from the loaded values files of a container
+ * @function
+ * @param {Object} params
+ * @param {Object} params.env - The current env of the task being run
+ * @param {Object} params.__internal - Any extra / injected params for injected taps
+ * @param {Object} params.containerRef - Reference to the container of the actions to load
+ *
+ * @returns {Object} - Found actions from the values files of the container 
+ */
+const getActionsFromValues = async params => {
+  let containerName = params.cmdContext ||
+    params.context ||
+    params.containerRef ||
+    params.container
+
+  containerName = !docker.isDockerId(containerName)
+    ? containerName
+    : await (async () => {
+        const container = await docker.container.get(containerName)
+        return container && container.name
+      })()
```

Comment:
Normally I would agree with you, but I just did a search in the code to see if this pattern is used anywhere else I didn't find it.
Typically we want the entire container object, which includes the name. It's not very often that we need name of the container and we don't already have it.

Even in this case 99.9% of the time we already have the name from the `params.cmdContext`
This code just covers a possible edge case where container ID is passed, but not a context. 
Which should never happen, because the only way we can get the container ID is if we already had the name

Really I could probably just delete it, but I figured might as well keep it just in case

Reply:
Sounds good. If it were me I would still refactor it just for readability, but I'm good with it either way in this case.